static/css/landingStyle.css

```diff
@@ -0,0 +1,268 @@
+
```

Comment:
We use boostrap instead of css,
boostrap is a library that wraps css

Reply:
I did use bootstrap, I just ran over it with css when needed,  in order to make the site more responsive and nice
see index.html the <link> tag in the head section 'imports' the bootstrap in my understanding

Reply:
Correct but you should use boostrap please refer this 
https://www.w3schools.com/bootstrap4/bootstrap_get_started.asp

You could play with and have immediate response for learning

Reply:
Yes, I understand, I will use bootstrap when needed. bootstrap + css is much more powerful imo
I also checked previous projects and saw that they used bootstrap and css